### PR TITLE
Production and testing diff output

### PR DESCRIPTION
**Pull request by ben851**
## What happens when your PR merges?

Adds workflows for Hasura and Blazer available through VPN in prod.

Also got the helmfile diff output to go to github actions comments!

## What are you changing?
- [ ] Releasing a new version of Notify
- [ ] Changing kubernetes configuration

## Provide some background on the changes
VPN

### CODE REVIEW

File: .github/workflows/helmfile_production_plan.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  PRODUCTION_AWS_ACCOUNT: ${{ secrets.PRODUCTION_AWS_ACCOUNT }}
+
+jobs:
+  helmfile-diff:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Inject token authentication
+        run: |
+          git config --global url."https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/".insteadOf "https://github.com/"
```
Comment (jimleroyer) [inline .github/workflows/helmfile_production_plan.yaml:22]: Oh interesting stuff going on here with the `insteadOf` instruction!

File: .github/workflows/helmfile_production_plan.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        id: helmfile_diff
+        run: |
+          echo 'var<<EOF' >> $GITHUB_OUTPUT
+          helmfile --environment staging diff >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
+      - name: Helmfile Diff Comment
+        uses: mshick/add-pr-comment@v2
+        with:
+          message: | 
+            Placeholder for helm diff once the output from GA works.
```
Comment (jimleroyer) [inline .github/workflows/helmfile_production_plan.yaml]: Is the diff issue resolved or it's ongoing?

File: .github/workflows/helmfile_staging_plan.yaml [hunk 3]
```diff
@@ -35,13 +35,19 @@ jobs:
...
+        run: |
+          echo 'var<<EOF' >> $GITHUB_OUTPUT
+          helmfile --environment staging diff >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
       - name: Helmfile Diff Comment
         uses: mshick/add-pr-comment@v2
         with:
           message: | 
-            Placeholder for helm diff once the output from GA works.
+            ${{join(steps.helmfile_diff.outputs.*, '\n')}}
```
Comment (jimleroyer) [inline .github/workflows/helmfile_staging_plan.yaml:53]: Oh that is the working diff here I assume?

**Review by jimleroyer — APPROVED** [review]
Looking like it's worth a shot.

File: .github/workflows/helmfile_production_plan.yaml (hunk 1 above)
Comment (ben851) [reply .github/workflows/helmfile_production_plan.yaml:22]: This is a copy paste from our other GA's

File: .github/workflows/helmfile_production_plan.yaml (hunk 2 above)
Comment (ben851) [reply .github/workflows/helmfile_production_plan.yaml]: Yeah I forgot to change prod - it's resolved!

File: .github/workflows/helmfile_staging_plan.yaml (hunk 3 above)
Comment (ben851) [reply .github/workflows/helmfile_staging_plan.yaml:53]: You got it!

**Pull request merged**